backend/src/fun_forecast_backend/main.py

```diff
@@ -91,3 +91,26 @@ async def get_activity_forecast(verb: str, lat: float, long: float) -> JSONRespo
         logger.error(f"Failed get_activity_forecast, inputs verb={verb}, lat={lat}, long={long}: {e}")
         empty_return = Forecast(verb=verb)
         return asdict(empty_return)
+
+@app.get("/api/v0/get-forecasts/{lat},{long}/{verbs}")
+async def get_activity_forecasts(lat: float, long: float, verbs: str) -> JSONResponse:
+    logger.debug(f"Processing get_activity_forecast request, inputs verbs={verbs}, lat={lat}, long={long}")
```

Comment:
Change these log lines, including line 110 and 114 to reflect the actual function name